### DISCUSSION TITLE
Rule "on DS18B20-3#Temperature>55.0 do ..." does not trigger

### DISCUSSION THREAD

**Discussion by dolocyl2** [arendst/Tasmota#22475] [Q&A] [answered]
Hi all
I created a simple Rule that should trigger if the measured temperature on DS18B20>55.0.
The rule should switch power on also switch rule2 on.
However it never triggers and I cannot figure out why. 
Maybe someone has a hint?

Here is what I see in console:
`10:49:54.397 MQT: tasmota/stat/tasmota_D754B3/RESULT = {"Rule1":{"State":"ON","Once":"OFF","StopOnError":"OFF","Length":62,"Free":449,"Rules":"on DS18B20-3#Temperature>55.0 do backlog Power1 1; rule2 1 end"}}`

`10:50:14.255 MQT: tasmota/tele/tasmota_D754B3/SENSOR = {"Time":"2024-11-13T10:50:14","Switch1":"ON","DS18B20-1":{"Id":"3C01B556381B","Temperature":44.1},"DS18B20-2":{"Id":"3C01B5566218","Temperature":35.3},"DS18B20-3":{"Id":"3C01B556BF0E","Temperature":56.6},"TempUnit":"C"}
`
So DS18B20-3 is >55.0 but rule1 does not trigger.
I am running on tasmota-release 14.3.0

**Comment by sfromis** [comment, marked as answer]
It usually helps to use correct syntax, like instead of that erroneous `end`

**Reply by dolocyl2** [reply]
Thank you for pointing that out.
I don't know where I've lost the "on" of the "endon" - and I sure did not notice looking at the rule many times...
It's working now, thanks!
